styleguide/components/bsp-gallery-fullscreen/interstitial1.html

```diff
@@ -0,0 +1,4 @@
+<!-- @todo: will need to get some definition on types of content, sizes, etc to account for -->
+<div style="width: 560px; margin: 0 auto; padding-top: 100px;">
```

Comment:
Can you replace this with a base interstitial slide CSS rule for display: flex and to align and center vertically and horizontally whatever is in there?